docker-compose-pool.yaml

```diff
@@ -14,6 +14,12 @@
 
 version: '3.5'
 
+volumes:
+  # Named volume for WPE to share MRENCLAVE with KME. If this has to be
+  # overridden, a parameter needs to be passed to KME enclave manager
```

Comment:
"If this has to be overridden ...." -> MRENCLAVE value can also be passed as a command line parameter (using the option --wpe_mr_enclave)  in WPE enclave manger. Then the MRENCLAVE value stored in file " wpe_mr_enclave.txt" in shared volume will be overridden by the value passed in command line.